internal/server/data/group.go

```diff
@@ -62,7 +62,9 @@ func DeleteGroups(db *gorm.DB, selectors ...SelectorFunc) error {
 
 func AddUsersToGroup(db *gorm.DB, groupID uid.ID, idsToAdd []uid.ID) error {
 	for _, id := range idsToAdd {
-		err := db.Exec("INSERT INTO identities_groups (group_id, identity_id) select ?, ? WHERE NOT EXISTS (SELECT 1 FROM identities_groups WHERE group_id = ? AND identity_id = ?)", groupID, id, groupID, id).Error
+		// This is effectively an "INSERT OR IGNORE" or "INSERT ... ON CONFLICT ... DO NOTHING" statement which
+		// works across both sqlite and postgres
```

Comment:
```suggestion
```

This should hopefully always be the case at the moment, so we don't need to state it

Reply:
My thinking on this is it's a non-standard-ish SQL call that could be replaced if we standardized on one database type.  If we only supported postgres, we should instead of "INSERT ... ON CONFLICT", but that won't work w/ sqlite.